src/pages/core/entrypoints/reply.mdx

```diff
@@ -0,0 +1,27 @@
+---
+tags: ["core", "entrypoints"]
+---
+
+import { Callout } from "nextra/components";
+
+# Reply
+
+The `reply` endpoint is another special one.
+
+It relates to CosmWasm's actor model which dictates how you call other contracts
+and interact with them.
+
+<Callout>TODO: Link to the actor model section once we have one</Callout>
+
+This endpoint gets invoked when the execution of an external contract you called
+finished. The reply parameter then lets you inspect and work with the response
```

Comment:
It's not necessarily just external contracts. It can be any message you send to the chain. For example, we used this in liquid staking before to execute code after a `DistributionMsg::WithdrawDelegatorReward`